play-services-core/src/main/kotlin/org/microg/gms/auth/credentials/assistedsignin/AssistedSignInService.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-FileCopyrightText: 2023 microG Project Team
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.microg.gms.auth.credentials.assistedsignin
+
+import com.google.android.gms.common.ConnectionResult
+import com.google.android.gms.common.Feature
+import com.google.android.gms.common.internal.ConnectionInfo
+import com.google.android.gms.common.internal.GetServiceRequest
+import com.google.android.gms.common.internal.IGmsCallbacks
+import org.microg.gms.BaseService
+import org.microg.gms.common.GmsService
+
+const val TAG = "AssistedSignInService"
+
+class AssistedSignInService : BaseService(TAG, GmsService.IDENTITY_SIGN_IN) {
+
+    override fun handleServiceRequest(callback: IGmsCallbacks, request: GetServiceRequest, service: GmsService) {
+        val connectionInfo = ConnectionInfo()
+        connectionInfo.features = arrayOf(
```

Comment:
This seems to be the feature set already defined in `org.microg.gms.auth.credentials.FEATURES`, so better don't declare it another time, so we don't have to update at two locations.

Reply:
you are right, I found it is the same